lib/rx-http.js

```diff
@@ -2,11 +2,12 @@ const httpRx = {};
 const request = require('request');
 const Observable = require('rxjs').Observable;
 const HTTP_STATUS_CODES = require('./http-status-codes');
+const HTTP_METHODS = require('./http-methods');
+const checkHttpMethod = require('./check-http-method');
```

Comment:
I propose to keep an original name of that variable in order to reflect semantic.
"Is<something>" implies a logical outcome (true or false).

Reply:
Do you mean it should be like
```
const isHttpMethodCorrect = require('./check-http-method');
```
?

Reply:
Yep.

Reply:
Done.